client/my-sites/site-settings/form-general.jsx

```diff
@@ -641,6 +659,8 @@ export class SiteSettingsFormGeneral extends Component {
 
 				{ this.props.isUnlaunchedSite ? this.renderLaunchSite() : this.privacySettings() }
 
+				{ this.toolbarOption() }
```

Comment:
Could we move that to the very bottom of the page, below the Footer Credit section? Just because it was at the bottom of the Writing page before.

Reply:
Okay, done